apps/site/test/site_web/views/trip_plan_view_test.exs

```diff
@@ -522,4 +522,62 @@ closest arrival to 12:00 AM, Thursday, January 1st."
       assert [{"input", _, _}] = Floki.find(html, ~s(input#plan-date-input[type="text"]))
     end
   end
+
+  describe "Fares logic" do
+    @fares_assigns %{
+      one_way_total: "$2.90",
+      round_trip_total: "$5.80"
+    }
+
+    test "renders fare information", %{conn: conn} do
+      html =
+        "_itinerary_fares.html"
+        |> render(Map.put(@fares_assigns, :conn, conn))
+        |> safe_to_string()
+
+      [{_, _, [one_way_fare]}, {_, _, [round_trip_fare]}] =
+        Floki.find(html, ".m-trip-plan-results__itinerary-fare")
+
+      assert one_way_fare == "$2.90 one way"
+      assert round_trip_fare == "$5.80 round trip"
+    end
+
+    test "gets the highest one-way fare" do
+      itinerary = %TripPlan.Itinerary{
+        start: nil,
+        stop: nil,
+        legs: [
+          %{
+            description: "BUS",
+            mode: %{
```

Comment:
Might make sense to add a non-transit leg to this test itinerary just to validate that the mode filtering doesn't produce unexpected results.